packages/a11y-speak/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "@wordpress/a11y-speak",
+  "version": "0.0.1",
+  "description": "Easily announce dynamic interface updates to screen readers using ARIA live regions.",
+  "homepage": "https://github.com/WordPress/packages/packages/a11y-speak/README.md",
```

Comment:
This will 404, needs the `blob/master` path fragment. Instead:

`https://github.com/WordPress/packages/tree/master/packages/a11y-speak`